ivy/functional/frontends/tensorflow/math.py

```diff
@@ -297,3 +297,20 @@ def zero_fraction(value, name="zero_fraction"):
 
 
 # TODO: Ibeta for Future Release
+
+
+def accumulate_n(inputs, shape=None, tensor_dtype=None, name="accumulate_n"):
+    tensor_dtype = tensor_dtype if tensor_dtype is not None else inputs[0].dtype
```

Comment:
Seems to be a bug with this line, based off of failing tensorflow frontend tests :)